config/config.go

```diff
@@ -28,6 +29,13 @@ type Config struct {
 	RDSDBInstance1             string `envconfig:"RDSINSTANCE1"`
 	RDSDBInstance2             string `envconfig:"RDSINSTANCE2"`
 	RDSDBInstance3             string `envconfig:"RDSINSTANCE3"`
+	// flag to use local postres instace provided by dp-compose
+	DPPostgresLocal            bool   `envconfig:"USEPOSTGRESLOCAL"`
```

Comment:
looks like we need these to for dev testing, can we see if we can existing environment variable with defaults configured here: https://github.com/ONSdigital/dp-areas-api/blob/develop/config/config.go#L55

Reply:
👍